File: Swift/프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.swift
func solution(_ s:String) -> String {
    var result = ""
    var wordIndex = -1
    
    for char in s {
        if char == " " {
            result.append(char)
            wordIndex = -1
        } else {
            wordIndex += 1
            if wordIndex % 2 == 0 {
                result.append(String(char).uppercased())
            } else {
                result.append(String(char).lowercased())
            }
        }
    }
    return result
}

File: Python3/프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.py
def solution(s):
    result = ""
    wordIndex = -1
    
    for char in s:
        if char == " ":
            result += char
            wordIndex = -1
        else:
            wordIndex += 1
            
            if wordIndex % 2 == 0:
                result += char.upper()
            else:
                result += char.lower()
                
    
    return result